pkg/recovery/recovery.go

```diff
@@ -45,12 +47,18 @@ type (
 )
 
 // CrashLog write down the current memory and stack info and the cpu/mem/disk infos into log dir
-func CrashLog(r interface{}, cfgLog log.GlobalConfig) {
+func CrashLog(r interface{}) {
 	log.S().Errorf("crashlog: %v", r)
-	if cfgLog.StderrRedirectFile != nil {
-		writeHeapProfile(filepath.Join(filepath.Dir(*cfgLog.StderrRedirectFile),
-			"heapdump_"+time.Now().Format("20060102150405")+".out"))
+	dir := config.Default.System.SystemLogDBPath
```

Comment:
hard coded dir is not recommended. If you believe this is a singleton, add a function `func SetCrashlogDir(dir string) error` to set and create crash log directory.

Reply:
it should **not** be `config.Default.System.SystemLogDBPath`, it should read the `SystemLogDBPath` from the `config` of server, same as other modules. So you can set the log file path by `config.yaml` file

Reply:
use a fixed directory to save generating log files.